Controller/SourceController.php

```diff
@@ -45,6 +45,12 @@ public function getAction($source) {
         $sourceConfig = $config['sources'][$source];
         /** @var AnalyticsInterface $analytics */
         $analytics = new $sourceConfig['class']($this->get('service_container'));
+        if (method_exists($analytics, 'setContext')) {
```

Comment:
method_exists will return true even for methods that you can't call (protected/private).
```
class A {
	private function foo() { }
	public function bar() { }
}

$a = new A();
var_dump(method_exists($a, 'foo'));
var_dump(is_callable([$a, 'foo']));
var_dump(method_exists($a, 'bar'));
var_dump(is_callable([$a, 'bar']));
```